packages/@o3r/extractors/src/utils/config-doc.ts

```diff
@@ -138,6 +141,17 @@ export function isO3rRequiredTagPresent(docText: string): boolean {
   return /@o3rRequired/.test(docText);
 }
 
+/**
+ * Get restriction keys from a given DocComment.
+ *
+ * The restriction keys extracted from @o3rRestrictionKey tag.
+ * @param docComment The DocComment to get category from
+ */
+export function getRestrictionKeysFromDocText(docComment: string): string[] {
+  return Array.from(docComment.matchAll(/@o3rRestrictionKey\s+(\w+|".*?"|'.*')$/gm))
```

Comment:
```suggestion
  return Array.from(docComment.matchAll(/@o3rRestrictionKey\s+(\w+|"[^"]*?"|'[^']*')$/gm))
```

To be a bit more "idiot proof" and avoid a `@o3rRestrictionKey "first" "second"`